spec/reek/configuration/configuration_file_finder_spec.rb

```diff
@@ -7,18 +7,19 @@
 
 RSpec.describe Reek::Configuration::ConfigurationFileFinder do
   describe '.find' do
-    it 'returns the path if it’s set' do
+    it 'returns the file in :path if it’s set' do
```

Comment:
Ah right, sorry, I’m a bit sick and not thinking clearly – it returns the `Pathname` to the file in the directory passed as the `path` argument. Maybe `it 'returns the configuration file found at the passed path'`? Technically, the `ConfigurationFileFinder` is also fuzzy (at it finds configuration files by returning `Pathnames`, not `Files`…).